docs/_docs/administrators-guide/lifecycle.adoc

```diff
@@ -40,7 +40,15 @@ First, the nodes specified in the `--cluster-management-group` argument form a R
 
 image::images/lifecycle1.png[Cluster initialization]
 
-Then, the nodes specified in the `--metastorage-group` argument form a RAFT group and take the role of *metastorage group*. These nodes will hold the authoritative copy of cluster meta information.
+Then, the nodes specified in the `--metastorage-group` argument form a RAFT group and take the role of *metastorage group*. These nodes will hold the authoritative copy of cluster meta information. If `--metastorage-group` is not specified, the nodes are chosen automatically. The number of nodes is chosen automatically as follows:
```

Comment:
We should also specify which nodes are chosen, that we chose by name in alphabetical order.

Reply:
```suggestion
Then, the nodes specified in the `--metastorage-group` argument form a RAFT group and take the role of *metastorage group*. These nodes will hold the authoritative copy of cluster meta information. If `--metastorage-group` is not specified, a set of nodes is automatically selected in alphabetical order. The number of selected nodes is determined as follows:
```

Reply:
Added in the latest version